konfera/models/event.py

```diff
@@ -53,6 +70,17 @@ def __str__(self):
     def duration(self):
         return (self.date_to - self.date_from).days + 1
 
+    @property
+    def cfp_open(self):
+        return self.cfp_allowed and self.cfp_end >= datetime.now(tz=timezone.utc)
```

Comment:
I think `timezone.now()` does the same as `datetime.now(tz=timezone.utc)` isn't it?